### ISSUE TITLE
Navbar:hover on tab

### ISSUE THREAD

**Issue by a19hu** [a19hu/SWC_2024#19]
when hover on text then show otherwise not .
Create a final navbar like this [navbar](http://swc.iitj.ac.in/)

![Screenshot from 2024-07-08 22-06-00](https://github.com/a19hu/SWC_2024/assets/132092345/bdbee597-3add-4b14-a158-f65f898f264a)

**Comment by krishchvn** [comment]
Hey. Can you assign the issue to me?

**Comment by krishchvn** [comment]
Hey, 
As per the final [navbar](http://swc.iitj.ac.in/), do you want to create exactly the same? 
(Btw, they have hover on div, not on text. Do you want to hover on text only?)

**Comment by a19hu** [comment]
> Hey, 
> As per the final [navbar](http://swc.iitj.ac.in/), do you want to create exactly the same? 
> (Btw, they have hover on div, not on text. Do you want to hover on text only?)

yes, you make it exactly same and other thing is hover on div then show otherwise not .

**Comment by a19hu** [comment]
@krishchvn are you working on this issue??

**Comment by krishchvn** [comment]
Yes. I'm sorry I didn't get time on weekdays. I'll update you today or tomorrow about the issue.

**Comment by krishchvn** [comment]
Your [final website](http://swc.iitj.ac.in/) isn't working.
I tried on two diff browsers. 1st image is chrome, 2nd is mozilla
![image](https://github.com/user-attachments/assets/c14dba80-a550-4e8d-8d76-aca91d829c7c)
![image](https://github.com/user-attachments/assets/20836970-44c3-48ae-8c1b-27a00fb46c49)

**Comment by a19hu** [comment]
@krishchvn Yes, there is some problem. For now I provide video for that.
[Screencast from 14-07-24 07:24:30 PM IST.webm](https://github.com/user-attachments/assets/4fd71ca8-d3fb-41f5-99c5-09ec0618b106)

**Comment by krishchvn** [comment]
Can you send me the video of responsive navbar? Also, what is the exact background color of navbar and home when active?
Also color when hover on links?

**Comment by a19hu** [comment]
> Can you send me the video of responsive navbar? Also, what is the exact background color of navbar and home when active?
> Also color when hover on links?

@krishchvn please check [website](http://swc.iitj.ac.in/)

**Comment by a19hu** [comment]
> > Can you send me the video of responsive navbar? Also, what is the exact background color of navbar and home when active?
> > Also color when hover on links?
> 
> @krishchvn please check [website](http://swc.iitj.ac.in/) 

if this is not work from here, please go Google and search "swc iitj" and first link open . then it is work .
